experimenter/experimenter/templates/changelog/overview.html

```diff
@@ -1,17 +1,44 @@
-{% extends "experimenter_base.html" %}
+{% extends "layout/layout_with_sidebar.html" %}
 
 {% block title %}
   History
 {% endblock %}
 
-{% block content %}
-    <h1>Changes Overview</h1>
-    <p>Experiment Slug: {{ experiment.slug }}</p>
-    {% for changelog in experiment.get_changelogs %}
-    <div>
-        <h2>{{ changelog.changed_on }}</h2>
-        <p><i class="fa-regular fa-pen-to-square"></i> : {{ changelog.message }}</p>  
-        <p><i class="fa-regular fa-user"></i> : {{ changelog.changed_by }}</p>
+
+{% block sidebar %}
+  {% include "common/experiment-details-sidebar.html" with experiment=experiment %}
```

Comment:
Today I learned the %include% tag could set variables!

Reply:
I stole it from the legacy code :)